### PR TITLE
17.17 -  Rewrite integration topic and fulltext concepts

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR does the following:

- Completely rewrites and renames the Integration topic, hopefully to make the content more clear and useful. Please review carefully!
- Moves some content out of indexes and into concept topics. This content probably hasn't been reviewed in a while, so please make sure it is still correct and applicable.

### CODE REVIEW

File: docs/search-views/fulltext-search/concepts/fulltext-search-setup.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Linking Collections
+
+When you create a search view, you choose which collections to link it to. A link transforms your collections into searchable data sources, creating a one-way data flow from a collection to a search view. By establishing multiple links from different collections to a single search view, you can perform a single search across multiple collections.
+
+You can also link more than one search view to a particular collection. Linking one collection to multiple search views allows you to apply different search strategies to the same data set. This flexibility is invaluable when dealing with complex applications with varied search requirements.
+
+## Handling Data Structures
+
+You can link key-value stores, document stores, and graph edge collections to search views. This setup allows your data to be treated as both flat and interconnected structures, supporting both straightforward attribute-based searches and more complex searches involving relationships between data entities.
```
Comment (maheshw2) [inline docs/search-views/fulltext-search/concepts/fulltext-search-setup.md:16]: Do we want to mention exception in `key-value stores, document stores, and graph edge collections`? If yes, then KV blob might be exception that @amolth-macrometa may confirm.

**Review by maheshw2 — APPROVED** [review]
Rest of the content looks good. Added a suggestion if it's relevant.

File: docs/search-views/fulltext-search/concepts/fulltext-search-setup.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/search-views/fulltext-search/concepts/fulltext-search-setup.md:16]: What do you mean by "exceptions"? Do you mean that blobs cannot be handled by fulltext search views, or are you referring to something else?

Comment (diana-macrometa) [reply docs/search-views/fulltext-search/concepts/fulltext-search-setup.md:16]: @maheshw2, see question above

Comment (maheshw2) [reply docs/search-views/fulltext-search/concepts/fulltext-search-setup.md:16]: @diana-macrometa yes, that is my understanding but it needs to be confirmed by @amolth-macrometa

Comment (amolth-macrometa) [reply docs/search-views/fulltext-search/concepts/fulltext-search-setup.md:16]: Correct; blob-based collections do not support search views of any kind.

Comment (diana-macrometa) [reply docs/search-views/fulltext-search/concepts/fulltext-search-setup.md:16]: Is the note I added below sufficient, or do I need more clarification?

**Comment by diana-macrometa** [comment]
Do we need any more changes to these topics?

File: docs/search-views/fulltext-search/concepts/fulltext-search-setup.md (hunk 1 above)
Comment (maheshw2) [reply docs/search-views/fulltext-search/concepts/fulltext-search-setup.md:16]: I think its good.

**Review by maheshw2 — APPROVED** [review]

**Pull request merged**